recipes/stripy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "stripy" %}
+{% set version = "v2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/underworldcode/{{ name|lower }}/archive/refs/tags/{{ version }}.tar.gz 
+  sha256: 6e40a36527757f99c5f1bb1220109c8e42281b274fef7fac6da1a8e8d240d67b 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - numpy 1.16.*
```

Comment:
This is wrong. See the first example at https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Is it? I wanted to ensure the minimum version would be 1.16, as in the Notes in the example.

Reply:
We don't support 1.16 anymore. Just `- numpy` would mean `1.17`.